charts/pega/values.yaml

```diff
@@ -41,6 +41,8 @@ global:
     driverUri: "YOUR_JDBC_DRIVER_URI"
     username: "YOUR_JDBC_USERNAME"
     password: "YOUR_JDBC_PASSWORD"
+    # Provide the below external secret name if JDBC_PASSWORD is configured from External Secrets Manager. Provide this parameter only if above password field is empty.
```

Comment:
# To avoid exposing a password, leave the jdbc.password parameter empty (no quotes), configure a JDBC password in the External Secrets Manager, and enter the external secret for the JDBC password.